acceptance/tests/environment/should_find_existing_production_environment.rb

```diff
@@ -0,0 +1,118 @@
+test_name "should find existing production environment"
+tag 'audit:high'
+
+require 'puppet/acceptance/i18ndemo_utils'
+extend Puppet::Acceptance::I18nDemoUtils
+
+agents.each do |agent|
+  initial_environment = on(agent, puppet("config print environment")).stdout.chomp
+  initial_environment_paths = on(agent, puppet("config print environmentpath")).stdout.chomp.split(File::PATH_SEPARATOR)
```

Comment:
one problem is that this ruby code runs on the coordinator node, so `File::PATH_SEPARATOR` is actually `:`. I think you can use https://github.com/puppetlabs/puppet/blob/e4b4578287a675bc53654cf62d11e4e04a16ae22/acceptance/lib/puppet/acceptance/module_utils.rb#L17 to get the array of module paths. Note the 'host' object has a `pathseparator` which comes from https://github.com/voxpupuli/beaker/blob/998112b326d6a3ed2e89b63605d302a4623788ee/lib/beaker/host/windows.rb#L23